pkg/secretless/plugin/connector/http/http.go

```diff
@@ -36,10 +46,16 @@ type Connector interface {
 // and you now have a Plugin. It does this by giving any function of the type
 // ConnectorConstructor a constructor method called NewConnector that simply
 // calls the function itself
-type ConnectorConstructor func (connector.Resources) Connector
+type ConnectorConstructor func(connector.Resources) Connector
+
+/*
+NewConnector returns a Connector, a one method interface that performs the actual
+authentication.
 
-// NewConnector is a ConnectorConstructor's implementation of the Plugin
-// interface. It simply delegates to the underlying function.
+When Secretless runs, it will call NewConnector once and then hold onto the
```

Comment:
When Secretless runs, it calls...........once and then holds onto the.................